subsys/bluetooth/audio/vocs.c

```diff
@@ -19,13 +19,16 @@
 
 #include "audio_internal.h"
 #include "vocs_internal.h"
+#include "zephyr/bluetooth/audio/audio.h"
 
 #define LOG_LEVEL CONFIG_BT_VOCS_LOG_LEVEL
 #include <zephyr/logging/log.h>
 LOG_MODULE_REGISTER(bt_vocs);
 
 #define VALID_VOCS_OPCODE(opcode)	((opcode) == BT_VOCS_OPCODE_SET_OFFSET)
 
+#define BT_AUDIO_LOCATION_RFU (~BT_AUDIO_LOCATION_ANY)
```

Comment:
Minor: We should probably avoid using the BT prefix for macros that only exist in specifics source files
```suggestion
#define AUDIO_LOCATION_RFU (~BT_AUDIO_LOCATION_ANY)
```

The alternative is to move this macro to `audio_internal`; quite possibly it's something we may use for other modules as well. Either way works :)